openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/Component/ProfilerSettingsModal.tsx

```diff
@@ -193,22 +193,41 @@ const ProfilerSettingsModal: React.FC<ProfilerSettingsModalProps> = ({
         <Col data-testid="profile-sample-container" span={24}>
           <p>Profile Sample %</p>
           <div className="tw-px-2 tw-mb-1.5">
-            <Slider
-              className="profiler-slider"
-              marks={{
-                0: '0%',
-                100: '100%',
-                [profileSample as number]: `${profileSample}%`,
-              }}
-              max={100}
-              min={0}
-              tooltipPlacement="bottom"
-              tooltipVisible={false}
-              value={profileSample}
-              onChange={(value) => {
-                setProfileSample(value);
-              }}
-            />
+            <Row>
+              <Col span={20}>
+                <Slider
+                  className="profiler-slider"
+                  marks={{
+                    0: '0%',
+                    100: '100%',
+                    [profileSample as number]: `${profileSample}%`,
+                  }}
+                  max={100}
+                  min={0}
+                  tooltipPlacement="bottom"
+                  tooltipVisible={false}
+                  value={profileSample}
+                  onChange={(value) => {
+                    setProfileSample(value);
+                  }}
+                />
+              </Col>
+              <Col span={4}>
+                <InputNumber
+                  formatter={(value) => `${value}%`}
+                  max={100}
+                  min={0}
+                  step={1}
+                  style={{
+                    margin: '0 16px',
+                  }}
```

Comment:
Please use gutter prop in Row for spacing!

Reply:
Updated